pkg/agent/testdata/AKSUbuntu1604+TempDisk+Containerd/line9.sh

```diff
@@ -73,6 +74,10 @@ ERR_AZURE_STACK_GET_SUBNET_PREFIX=122
 ERR_SWAP_CREAT_FAIL=130 
 ERR_SWAP_CREAT_INSUFFICIENT_DISK_SPACE=131 
 
+ERR_TELEPORTD_PLUGIN_URL_NOT_SPECIFIED=150 
```

Comment:
super nit: we should be conservative in using error code as there are only limited number of them. consider merge this to download error and just print out an error message in cse